play-json/src/main/scala/play/api/libs/json/JsConstraints.scala

```diff
@@ -16,9 +16,12 @@ trait PathFormat {
   def at[A](path: JsPath)(implicit f: Format[A]): OFormat[A] =
     OFormat[A](Reads.at(path)(f), Writes.at(path)(f))
 
-  def nullable[A](path: JsPath)(implicit f: Format[A]): OFormat[Option[A]] =
+   def nullable[A](path: JsPath)(implicit f: Format[A]): OFormat[Option[A]] =
     OFormat(Reads.nullable(path)(f), Writes.nullable(path)(f))
 
+  def optional[A](path: JsPath, defaultValue: => Option[A])(implicit f: Format[A]): OFormat[Option[A]] =
```

Comment:
Either using previous naming, or a different one which would correspond to the semantic "nullable-with-default"